# ~/catkin_ws/src/car_tracker_yolo/msg/Detection2DArray.msg
# Represents an array of 2D object detections

std_msgs/Header header
car_tracker_yolo/Detection2D[] detections